SortWoSTL/namesSortWoSTL.cpp

```diff
@@ -0,0 +1,98 @@
+#include "namesSortWoSTL.h"
+#include <iostream>
+#include <fstream>
+#include <string>
+#include <algorithm>
+#include <stdio.h>
+
+//constructor//
+NamesSortWoSTL::NamesSortWoSTL(){}
+
+//Writing in file 5k names//
+void NamesSortWoSTL::AddToFile(){
+    std::string arr[81] = {"Peter ", "Katya ", "Ann ", "Steve ", "John "};
+    std::ofstream fileo("/home/ngb/Desktop/C++/HW/Names/name");
+    if(fileo.is_open()){
+        for(int i = 0; i < 1000; i++){
+            for (int j = 0; j < 5; j++){
+                fileo << arr[j];
+            }
+        }
+        std::cout <<std::endl;
+    }
+    else {
+        std::cout << "Can't write data" << std::endl;
+    }
+    fileo.close();
+}
+
+//writing names from file to string array//
+    std::string NamesSortWoSTL::ReadFromFile(){
+    std::ifstream file("/home/ngb/Desktop/C++/HW/Names/name");
+    if (file.is_open()){
+        for (int i = 0; i < size-1; i++){
+        file >> temp;
+        names[i] = temp;
+        }
+    }
+    else{
+        std::cout << "Something gone wrong!" << std::endl;
+    }
+    file.close();
+    return *names;
+}
+    //writing to char array for calculate weight//
+    char NamesSortWoSTL::WriteToArray(){
+        std::ifstream file("/home/ngb/Desktop/C++/HW/Names/name");
+        if (file.is_open()){
+            file.get(ascii,len);
+        }
+        else{
+            std::cout << "Something gone wrong!" << std::endl;
+        }
+        file.close();
+        return *ascii;
+    }
+
+//sorting//
+void NamesSortWoSTL::Sort(){
+
+    std::sort(std::begin(names),std::end(names));
```

Comment:
well, it's great you've found this library method, but the task was to actually implement it yourself$))